### DISCUSSION TITLE
Toggle for normalisation of Accuracy Heatmap

### DISCUSSION THREAD

**Discussion by Bubbleman532** [ppy/osu#29518] [Ideas]
![image](https://github.com/user-attachments/assets/dd926dbb-0c6d-4d2f-a60f-9328b23192c4)

It would be nice to have a button on this element allowing the toggle of normalisation for this feature - this would make checking whether there's common issue of misaiming in a certain direction (always missing because too far left, for example) much easier to see at a glance.

**Comment by vatei** [comment]
Is it not already normalized? 
From this comment that's how I understood it anyway https://github.com/ppy/osu/issues/11579#issuecomment-766172061
This thread also mentioned that they are against a toggle I think

**Reply by peppy** [reply]
Yeah I'm not sure I understand the implication of normalised here. You want to see if a user is constantly aiming to the left of the screen relative to the beatmap? Like to see if someone has an issue where their brain and eyes are de-synced in an axis?

**Reply by Bubbleman532** [reply]
Yeah it's already normalised, what I'm asking for is a button that removes the normalisation. This doesn't have many implications but does let you tell if you need to move your tablet laterally on your table - sometimes my only misses are to the left of notes and that means i just need to move my tablet to fix everything

**Reply by peppy** [reply]
I wonder if we should just show both versions rather than have a toggle.

**Reply by bdach** [reply]
Full disclosure that I was approached about this in DMs before this discussion was posted, and

> I wonder if we should just show both versions rather than have a toggle.

I said as much in that conversation, so that's a +1 from me for sure.
